frontend/static/css/index.css

```diff
@@ -0,0 +1,119 @@
+body {
+    height: 100%;
+    margin: 0;
+  }
+
+.topnav {
+    /* overflow: hidden; */
+    /* background-color: rgb(148, 148, 148); */
+    background-color: #3b694483;
+    position: fixed; /* Set the navbar to fixed position */
+    top: 0; /* Position the navbar at the top of the page */
+    left: 0;
+    width: 100%; /* Full width */
+    z-index:1000;/*add z-index*/
+}
+
+.topnav a {
+    float: left;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    font-size: 20px;
+}
+
+.topnav a.active {
+    background-color: #8cee8786;
+    color: rgb(92, 97, 54);
+}
+
+.topnav .icon {
+    display: none;
+}
+
+/* The dropdown container */
+.dropdown {
+    float: left;
+    overflow: hidden;
+}
+  
+/* Dropdown button */
+.dropdown .dropbtn {
+    font-size: 20px;
+    border: none;
+    outline: none;
+    color: white;
+    padding: 14px 16px;
+    background-color: inherit;
+    font-family: inherit; /* Important for vertical align on mobile phones */
+    margin: 0; /* Important for vertical align on mobile phones */
+    /* z-index:1000; */
+}
+
+  /* Show the dropdown menu on hover */
+.topnav a:hover, .dropdown:hover, .dropbtn:hover{
```

Comment:
The commas between elements/objects, are for the properties to be always applicable.